integration/kubernetes/k8s-init.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="test-init"
+	get_pod_config_dir
+}
+
+@test "Test init containers" {
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-init.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
```

Comment:
In case there is really a bug this can wait forever, which seems the behaviour of `kubectl wait` if `timeout` is not passed. Or does the caller of this test sets a timeout?

Reply:
@wainersm it has a timeout

Reply:
ah, ok, thanks @GabyCT